second-gen/packages/swc/package.json

```diff
@@ -49,33 +49,33 @@
         "lit": "^2.5.0 || ^3.1.3"
     },
     "devDependencies": {
-        "@axe-core/playwright": "^4.10.2",
-        "@custom-elements-manifest/analyzer": "^0.10.5",
+        "@axe-core/playwright": "4.10.2",
+        "@custom-elements-manifest/analyzer": "0.10.8",
         "@storybook/addon-a11y": "9.1.10",
         "@storybook/addon-designs": "10.0.2",
         "@storybook/addon-docs": "9.1.10",
-        "@storybook/addon-vitest": "^9.1.10",
-        "@storybook/test-runner": "^0.23.0",
+        "@storybook/addon-vitest": "9.1.10",
+        "@storybook/test-runner": "0.23.0",
         "@storybook/web-components-vite": "9.1.10",
-        "@types/react": "^19",
-        "@types/react-dom": "^19",
-        "@vitest/browser": "^3.2.4",
+        "@types/react": "19.1.16",
+        "@types/react-dom": "19.1.9",
+        "@vitest/browser": "3.2.4",
         "@vitest/coverage-v8": "3.2.4",
-        "@vitest/ui": "^3.2.4",
-        "@wc-toolkit/storybook-helpers": "^9.0.1",
-        "autoprefixer": "^10.4.21",
-        "glob": "^11.0.0",
-        "playwright": "^1.55.0",
-        "postcss": "^8.5.6",
-        "postcss-preset-env": "^10.3.1",
-        "react": "^19.1.1",
-        "react-dom": "^19.1.1",
-        "rimraf": "^6.0.1",
-        "storybook": "^9.1.10",
-        "typescript": "^5.3.3",
-        "vite": "^7.1.3",
-        "vite-plugin-dts": "^4.3.0",
-        "vite-plugin-lit-css": "^2.2.1",
+        "@vitest/ui": "3.2.4",
+        "@wc-toolkit/storybook-helpers": "9.0.1",
+        "autoprefixer": "10.4.21",
+        "glob": "11.0.3",
+        "playwright": "1.53.1",
+        "postcss": "8.5.6",
+        "postcss-preset-env": "10.4.0",
+        "react": "19.1.1",
+        "react-dom": "19.1.1",
+        "rimraf": "6.0.1",
+        "storybook": "9.1.10",
+        "typescript": "5.9.2",
```

Comment:
This change needs a full TS build to check for regression. Laying this out here for other reviewers